modules/api/docs/LorisRESTAPI.md

```diff
@@ -787,82 +787,184 @@ The JSON object is of the form:
                     "InversionTime" : NULL,
                     "SliceThickness" : "3.5",
                     "Modality" : "MR",
                     "SeriesUID" : "3.4.5.6.1507",
-                }]
+                }
+            ]
         },
         {
             "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-def456.tar",
-            "SeriesInfo" :
-                [{
-                "SeriesDescription" : "MPRAGE_ipat2",
-                "SeriesNumber" : "2",
-                "EchoTime" : "2.98",
-                "RepetitionTime" : "2300",
-                "InversionTime" : "900",
-                "SliceThickness" : "1",
-                "Modality" : "MR",
-                "SeriesUID" : "1.7.8.9.1296",
-                }]
+            "SeriesInfo" : [
+                {
+                  "SeriesDescription" : "MPRAGE_ipat2",
+                  "SeriesNumber" : 2,
+                  "EchoTime" : "2.98",
+                  "RepetitionTime" : "2300",
+                  "InversionTime" : "900",
+                  "SliceThickness" : "1",
+                  "Modality" : "MR",
+                  "SeriesUID" : "1.7.8.9.1296",
+                }
+            ]
         }
-    ]    
+    ]
 }
 ```
 
-The `Modality` header in the SeriesInfo is either `MR` or `PT` for MRI or PET 
+The `Modality` header in the SeriesInfo is either `MR` or `PT` for MRI or PET
 scans, respectively.
 
+```
+POST /candidates/$CandID/$VisitLabel/dicoms
+```
+
+The body of the POST request should be encoded as a form data and contain a
+field `File` with the imaging file, and `Json`, with the following keys and
+values:
+
+```js
+{
+    "CandID": $CandID,
+    "PSCID": $PSCID,
+    "VisitLabel": $VisitLabel,
+    "IsPhantom": boolean
+}
+```
+
+The JSON can optionally have a boolean `Overwrite` attribute that can be used
+to overwrite an existing file (`false` if not present).
+A successful request will be answered by a `303 See Other` response with its
+`Location` header pointing to the processes list of the new upload.
+(See 5.3 Tar Level processes)
+
 ### 5.2 Tar Level Data
+
+This section describe how to upload DICOM studies and how to start and monitor
+`mri_upload` processes.
+DICOM studies that have been successfully uploaded and processed can be downloaded with
+the following `GET` request:
+
 ```
-GET /candidates/$CandID/$VisitLabel/dicoms/$Tarname
+GET /candidates/$CandID/$VisitLabel/dicoms/$ArchiveName
 ```
 
-Returns/Downloads a `tar` file which contains a `.meta` and a `.log` text 
+Returns/Downloads a `tar` file which contains a `.meta` and a `.log` text
 files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
-scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
+scanning session, and as retrieved from `/candidates/$CandID/$VisitLabel/dicoms`.
 
-Only `GET` is currently supported.
+To get a list of the processes and their status for a given DICOM study previously uploaded use the following:
```

Comment:
This is a different endpoint, seems to be missing a subheader.

Reply:
https://github.com/aces/Loris/pull/9154/commits/84b63983e9e4f3eec80c3940dac0e0029559d755

Reply:
https://github.com/aces/Loris/pull/9154/commits/8230e890555a37af5a6301f4240e43172988c1a6

This commit removes the documentation from the API v0.0.3. In practice, the new endpoints were already v0.0.4-only, but there are a few v0.0.4 POST routes that are added to v0.0.3 endpoints, not sure what do do here (ignore or implement method specific supported versions ?).

Reply:
Subheaders v2:

https://github.com/aces/Loris/pull/9154/commits/0c521249d1c16dd5beb965f191d4fc17c4818715